app/Models/Image.php

```diff
@@ -7,9 +7,9 @@
 class Image extends Model
 {
     function userImage(){
-        $this->belongsTo(User::class, 'id', 'imgable_id');
+        return $this->belongsTo(User::class, 'id', 'imgable_id');
     }
     function imageCourse(){
-        $this->belongsTo(Course::class, 'id', 'imgable_id');
+        return $this->belongsTo(Course::class, 'id', 'imgable_id');
     }
 }
```

Comment:
đặt tên sai quy tắc
https://laravel.com/docs/8.x/eloquent-relationships#one-to-many